src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -320,36 +329,36 @@ real64 SolverBase::setNextDt( real64 const & currentDt,
   {
     if( nextDtNewton > currentDt )
     {
-      GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "{}: time-step required will be increased based on number of iterations.",
-                                          getName() ) );
+      GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::TimeStep, GEOS_FMT( "{}: time-step required will be increased based on number of iterations.",
```

Comment:
Did you forget the messages line 323 & 326 or is it on purpose?
If it is a mistake, please verify that there is no other ones in your targeted classes.

Reply:
It would be part of a future PR, it concerning nonlinearSolverParameter